src/core/utils/balance.ts

```diff
@@ -15,8 +15,13 @@ export const calculateBalance = (
     const tokenKeys = Object.keys((account?.tokens || {})[chainId] || {});
     let balance = new BigNumber(0);
 
-    tokenKeys.map(key => {
-        const token = account.tokens[chainId][key];
+    const blockchainCoinTokenConfig = getTokenConfig(
```

Comment:
After `if (blockchain ===)` now its `getTokenConfig` inside core
Its ok for now